pkg/cmd/repo/rename/rename.go

```diff
@@ -49,9 +49,27 @@ func NewCmdRename(f *cmdutil.Factory, runf func(*RenameOptions) error) *cobra.Co
 	cmd := &cobra.Command{
 		Use:   "rename [<new-name>]",
 		Short: "Rename a repository",
-		Long: heredoc.Doc(`Rename a GitHub repository.
-
-		By default, this renames the current repository; otherwise renames the specified repository.`),
+		Long: heredoc.Docf(`
+			Rename a GitHub repository.
+			
+			%[1]s<new-name>%[1]s is the desired repository name without the owner.
+			
+			By default, the current repository is renamed. Otherwise, the repository specified
+			with %[1]s--repo%[1]s is renamed.
+			
+			To transfer repository ownership to another user account or organization,
+			you must follow additional steps on GitHub.com
+
+			For more information on transferring repository ownership, see:
+			<https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository>
```

Comment:
I don't love putting a long docs link here 🫤  - would like feedback on this. It doesn't seem like something we do typically. 

**Potential options:**
- Remove the link
- Shorten the link
- Keep it as-is

Reply:
Don't mind. It exists in other places already like `pr list`, `search code`. On the other hand `repo edit` uses `gh.io` short link.

Reply:
I guess we can leave it as-is then? If we're doing it in other places, it sounds like it should be fine?

Reply:
Either way is fine: we can keep the long url or shorten it.

Reply:
I don't know if we want another shortlink to maintain if we can avoid it 🤔 As I think about this more, I think I'm more in favor of leaving it expanded, unless someone objects.